sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/TelemetryOptions.cs

```diff
@@ -8,6 +8,9 @@ namespace Azure.DigitalTwins.Core.Models
     /// <summary>
     /// The additional information to be used when processing a telemetry request.
     /// </summary>
+    /// <remarks>
+    /// For more samples and information, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
```

Comment:
I don't feel 100% on this wording, so if you'd like to propose something better, I'm all ears! 🐰

Reply:
I feel it is sufficient to say - For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples"/>